src/atoms/cc-toggle.js

```diff
@@ -47,6 +47,7 @@ export class CcToggle extends LitElement {
       choices: { type: Array },
       disabled: { type: Boolean },
       hideText: { type: Boolean, attribute: 'hide-text' },
+      inline: { type: Boolean, reflect: true },
```

Comment:
Ignore the whole commit right now. I'd rather discuss this in sync as the `<fieldset>` / `<legend>` styling is quite tricky :roll_eyes: